pkg/cnservice/server.go

```diff
@@ -483,6 +485,23 @@ func (s *service) getTxnClient() (c client.TxnClient, err error) {
 			opts = append(opts,
 				client.WithEnableRefreshExpression())
 		}
+		if !s.cfg.Txn.DisableLeakCheck {
```

Comment:
I would have used EnableLeakCheck, why double negation?

Reply:
Or simply use MaxActiverAges > 0

Reply:
because i want to enable leak checker as default, so use disableLeakCheck as config, so no config file need changed. If use maxActiveAges to check is leak check is enabled, all test config need to changed.